packages/common/src/lib/entity/entity-table/entity-table.component.ts

```diff
@@ -692,7 +694,8 @@ export class EntityTableComponent implements OnInit, OnChanges, OnDestroy {
         value = entity?.properties[this.getColumnKeyWithoutPropertiesTag(column.linkColumnForce)];
       } else {
         column.domainValues.forEach(option => {
-          if (typeof value === 'string' && /^\d+$/.test(value)) {
+          if (typeof value === 'string' && /^\d+$/.test(value) &&
+            (value.length > 1 && value.charAt(0) !== '0')) {
```

Comment:
On pourrait être plus générique pour faciliter la maintenance.
```suggestion
          if (this.isValidNumber(value)) {
             ....
          }
          
          private isValidNumber(value: unknown): boolean {
              return typeof value === 'string' && /^\d+$/.test(value) && (value.length > 1 && value.charAt(0) !== '0');
          }
```

Reply:
Un petit commentaire sinon c'est bien beau, good job

Reply:
Oui ça fait du sens, je ne sais même pas pourquoi je n'y ai pas pensé... 😪 Je corrige ça